content/docs/user-guide/components/reference/gradients/image-gradient.md

```diff
@@ -21,23 +22,53 @@ Add the **Image Gradient** component to generate a gradient from an image, with
+
+### Advanced properties
+
+The **Advanced** group of properties has an in-line toggle switch that enables/disables the entire group of advanced properties. By default, these are disabled.
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Channel To Use** | The channel component of the image to sample from.<br><br>The `Terrarium` option is for an image-based terrain file format as defined here: https://www.mapzen.com/blog/terrain-tile-service/ | `Red`, `Green`, `Blue`, `Alpha`, `Terrarium` | `Red` |
+| **Custom Scale** | Choose a scaling operation to be applied to all image data.<br><br>The `Auto` option will automatically scale based on the min/max values in the image data. | `None`, `Auto`, `Manual` | `None` |
+| **Range Minimum** | The minimum value used to scale image data into the `0`-`1` range for the gradient. All image values at or below the minimum will be scaled to `0`.<br> <br>*This field is available only if the **Custom Scale** field is set to `Manual`.* | Float: `0.0` to `1.0`  | `0.0` |
+| **Range Maximum** | The maximum value used to scale image data into the `0`-`1` range for the gradient. All image values at or above the maximum will be scaled to `1`.<br> <br>*This field is available only if the **Custom Scale** field is set to `Manual`.* | Float: `0.0` to `1.0`  | `1.0` |
+| **Mip Index** | Specify which mip level index to sample from.<br><br>If you specify a mip level higher than the number of mip levels available in your image, then the highest existing mip index will be used. | Integer: `0` to `AZ::RHI::Limits::Image::MipCountMax` (15) | `0` |
+| **Sampling Type** | The sampling type to be used on the image data. | `Point`, `Bilinear` | `Point` |
+
+### Sampling types
+
+There are several sampling types to choose from. To illustrate, here are previews of the same source image using the
+different supported sampling types.
+
+#### Point
+
+`Point` sampling is the default and only samples a single point for each pixel of the source image.
+
+![Image Gradient using point sampling](/images/user-guide/components/reference/gradients/image-gradient-component-point.png)
+
+#### Bilinear
+
+The `Bilinear` filter can be used to smooth out the image by requesting four points in a grid around a requested
+pixel and then performing interpolation between the points.
+
+![Image Gradient using bilinear interpolation](/images/user-guide/components/reference/gradients/image-gradient-component-bilinear.png)
 
 ## ImageGradientRequestBus
 
 Use the following request functions with the `ImageGradientRequestBus` EBus interface to communicate with Image Gradient components in your game.
 
 | Request Name | Description | Parameter | Return | Scriptable |
 |-|-|-|-|-|
-| `GetImageAssetPath` | Returns the value of the **Image Asset** property. | None | Path: String | Yes |
+| `GetImageAssetPath` | Returns the path of the **AZ::RPI::StreamingImageAsset** property. | None | Path: String | Yes |
```

Comment:
I know this formatting was here before, but code should be in in-line code style, like `AZ::RPI::StreamingImageAsset`. Would help if you can update the remaining ones in this PR too - not a big deal if not. 
```suggestion
| `GetImageAssetPath` | Returns the path of the `AZ::RPI::StreamingImageAsset` property. | None | String | Yes |
```

Reply:
Sure thing!